### PR TITLE
Plugins: Update plugin signing copy + docs

### PR DESCRIPTION
**Pull request by wbrowne**
As of V8.0, we no longer distinguish between backend vs frontend plugins that are unsigned - they are both considered equal

### CODE REVIEW

**Review by jackw — APPROVED** [review]
LGTM! 🚀 

Maybe worth waiting for an approval from the content team before merging?

**Comment by oddlittlebird** [comment]
> Maybe worth waiting for an approval from the content team before merging?

Yes, it is. The Docs Squad appreciates it. :)

And in this case, if you have time, I would like @marcusolsson to take a look and weigh in. He is heavily involved with our plugin community and might have more ideas of what needs to be updated.

File: docs/sources/plugins/plugin-signatures.md
Comment (oddlittlebird) [inline docs/sources/plugins/plugin-signatures.md:19]: This seems like an odd place for this statement. Isn't it assumed that we sign our own plugins? And if we're going to say it, I would recommend it be up in the prose, not in a note. Perhaps a paragraph leading in to the one in line 21.

File: public/app/features/plugins/PluginsErrorsInfo.tsx
Comment (oddlittlebird) [inline public/app/features/plugins/PluginsErrorsInfo.tsx:50]: "encountered" seems odd here. Wouldn't it be something about "You attempted to load unsigned plugins" or "You have opened Grafana with unsigned plugins installed.""?

For the second sentence, I would reverse the order.

Grafana Labs cannot guarantee the integrity of unsigned plugins. We recommend only using signed plugins.

**Review by oddlittlebird — COMMENTED** [review]
Added a few comments, leaving approval for @marcusolsson or @achatterjee-grafana

File: docs/sources/plugins/plugin-signatures.md
Comment (marcusolsson) [reply docs/sources/plugins/plugin-signatures.md:19]: I agree. It feels like it should be implied that we sign our own plugins. Would it make more sense to invert this statement to list the Grafana Labs authored plugins that aren't (but should be) signed?

**Review by marcusolsson — APPROVED** [review]

File: docs/sources/plugins/plugin-signatures.md
Comment (wbrowne) [reply docs/sources/plugins/plugin-signatures.md:19]: I'd be happy to completely omit it if that works too?

>Would it make more sense to invert this statement to list the Grafana Labs authored plugins that aren't (but should be) signed?

Are there many of these? And will they exist like that for a long time?

File: public/app/features/plugins/PluginsErrorsInfo.tsx
Comment (wbrowne) [reply public/app/features/plugins/PluginsErrorsInfo.tsx:50]: It's a little more nuanced I suppose. So Grafana will try to load all plugins within the relevant plugin folder - and users can put whatever they want to in there. This warning will show them the list of plugins that Grafana found to be unsigned (not having a signature at all or having a bad signature). That's why it's more like _"hey Grafana found these things.."_

**Comment by oddlittlebird** [comment]
LGTM, but @achatterjee-grafana should have a final look and approve.

**Comment by wbrowne** [comment]
@achatterjee-grafana WDYT?

File: docs/sources/plugins/catalog.md
Comment (osg-grafana) [inline docs/sources/plugins/catalog.md:9]:
```suggestion
The plugin catalog allows you to browse and manage plugins from within Grafana. Only a Grafana server administrator can access and use the catalog.
```

File: docs/sources/plugins/plugin-signatures.md
Comment (osg-grafana) [inline docs/sources/plugins/plugin-signatures.md:11]:
```suggestion
At startup, Grafana verifies the signatures of every plugin in the plugin directory. If a plugin is unsigned, then Grafana does not load nor start it. To see the result of this verification for each plugin, navigate to **Configuration** > **Plugins**.
```

File: docs/sources/plugins/plugin-signatures.md
Comment (osg-grafana) [inline docs/sources/plugins/plugin-signatures.md:16]:
```suggestion
WARN[05-26|12:00:00] Some plugin scanning errors were found     errors="plugin '<plugin id>' is unsigned, plugin '<plugin id>' has an invalid signature"
```
What is up with the formatting here?

File: docs/sources/plugins/plugin-signatures.md
Comment (osg-grafana) [inline docs/sources/plugins/plugin-signatures.md:19]:
```suggestion
If you are a plugin developer and want to know how to sign your plugin, refer to [Sign a plugin]({{< relref "../developers/plugins/sign-a-plugin.md" >}}).
```

File: docs/sources/plugins/plugin-signatures.md
Comment (osg-grafana) [inline docs/sources/plugins/plugin-signatures.md:46]:
```suggestion
WARN[06-01|16:45:59] Running an unsigned plugin pluginID=<plugin id>
```

**Review by osg-grafana — APPROVED** [review]
Approving, and please consider input despite the approval.

**Comment by osg-grafana** [comment]
Note: I provided feedback about this PR because the folks in the US are on holiday today.

**Pull request merged**
